synth.py

```diff
@@ -38,6 +38,10 @@
         'cloud.google.comgrafeas.io',
         'grafeas.io')
 
+s.replace('src/v1/*.js',
```

Comment:
Guessing this is a bug in the generator or the protos?  Please file a bug in `googleapis/gapic-generator`, and link to the issue in the synth file :)

Reply:
we've patched this internally now :+1: